core/src/main/java/com/scalar/db/common/error/CoreError.java

```diff
@@ -647,10 +647,10 @@ public enum CoreError implements ScalarDbError {
           + "If you want to modify a condition, please use clearConditions() to remove all existing conditions first",
       "",
       ""),
-  TRANSPARENT_DATA_ENCRYPTION_NOT_ENABLED(
+  ENCRYPTION_NOT_ENABLED(
```

Comment:
@josh-wong I’ve updated the enum constant name here in commit c4ae68e23bc1debb94cf5d5e971387ae1c8d9b7b.

Reply:
Great, thank you! I wasn't confident about changing anything besides error messages since I was worried it could break something😅